ocs_ci/framework/conf/default_config.yaml

```diff
@@ -79,6 +79,11 @@ DEPLOYMENT:
   # Following options are defaults for the min and max Noobaa endpoints
   min_noobaa_endpoints: 2
   max_noobaa_endpoints: 3
+  # The subnet prefix length to assign to each individual node.
+  # For example, if hostPrefix is set to 23, then each node is assigned a /23 subnet out of the given cidr,
+  # which allows for 510 (2^(32 - 23) - 2) pod IPs addresses.
+  # https://docs.openshift.com/container-platform/4.1/installing/installing_bare_metal/installing-bare-metal.html
+  cluster_host_prefix: 23
```

Comment:
The install config is rendered with the `ENV_DATA` as you can [see here](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/deployment/ocp.py#L134-L136).

This will have to move to the `ENV_DATA` section before it will be applied to the install configs when they are rendered.

Reply:
Thanks for comment @clacroix12  updated the code accordingly, now I get why observed deployment issues.